src/index.html

```diff
@@ -16,7 +16,107 @@
       href="./style.css"
     />
   </head>
+  <link
```

Comment:
The 'link' tag for fonts should be inside the 'head' tag. It's a good practice to put all links and meta tags inside the 'head' tag.